lms/djangoapps/courseware/views/views.py

```diff
@@ -1714,12 +1715,10 @@ def _get_fa_header(header):
 
 
 FA_INCOME_LABEL = _('Annual Household Income')
```

Comment:
Perhaps consider bringing this one in-line with the `ugettext_noop` method instead of being re-hardcoded in the zendesk submission below; I _wouldn't_ necessarily make all the other fields in the zendesk submission work this way, but this one seems _so_ similar to the cases you updated in this PR that I'd make it work the same way.  (I leave it to your discretion and/or @emma-green's input, as this PR looks good)

Reply:
That's a good call, meant to do this in a separate commit but forgot, updating now!

Reply:
It's one of those cases of "don't fix unless it's broken" vs. making the code clean

Reply:
Yup, that balance is why I'd do this one but not others 👍